packages/portfolio-contract/test/evm-chain.exo.test.ts

```diff
@@ -0,0 +1,353 @@
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { makeHeapZone } from '@agoric/zone';
+import { prepareVowTools } from '@agoric/vow';
+import { Far } from '@endo/marshal';
+import { prepareEvmChainKit } from '../src/evm-chain.exo.ts';
+
+// Minimal GMPAccountInfo stub
+const makeAccountInfo = (n: number) => ({
+  namespace: 'eip155' as const,
+  chainName: 'arbitrum' as any, // AxelarChain stub
+  chainId: `eip155:42161:${n}` as any,
+  remoteAddress:
+    `0xdeadbeef${n.toString(16).padStart(2, '0')}` as `0x${string}`,
+});
+
+// Remotable gmpChain stub to satisfy passability when included in provision context
+const makeFakeGmpChain = () =>
+  Far('FakeGmpChain', {
+    getChainId: () => ({ chainId: '42161' }),
+  });
+
+// Minimal LocalAccount stub satisfying type via cast
+const makeFakeLocalAccount = () => {
+  // Declare as Remotable to satisfy passability requirements when methods are
+  // passed through provisioning contexts.
+  // We keep methods minimal and no-op.
+  const acct: any = Far('FakeLocalAccount', {
+    getAddress: () => ({ value: 'cosmos:feeacct' }),
+    send: async () => {},
+    getBalance: async () => 0n,
+    getBalances: async () => [],
+    sendAll: async () => {},
+    transfer: async () => {},
+    makeChild: () => acct,
+  });
+  return acct as any;
```

Comment:
Feel free to stub or even omit methods, but `any` is likely to hide problems that we like to see:

```suggestion
  // Declare as Remotable to satisfy passability requirements when methods are
  // passed through provisioning contexts.
  // We keep methods minimal and no-op.
  const acct = Far('FakeLocalAccount', {
    getAddress: () => ({ value: 'cosmos:feeacct' }),
  }) as unknown as LocalAccount;
  return acct;
```